.github/workflows/code-coverage.yml

```diff
@@ -77,3 +77,84 @@ jobs:
 
     - uses: ammaraskar/gcc-problem-matcher@0.2.0
       name: GCC Problem Matcher
+    
+  mingw-wine-coverage:
+    runs-on: ubuntu-24.04
+    timeout-minutes: 50
+    env:
+      WINEARCH: win64
+      WINEPATH: Z:\usr\lib\gcc\x86_64-w64-mingw32\13-posix
+      WINEPREFIX: /home/runner/.wine64
+      TZ: UTC
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install packages for MinGW + Wine
+        run: |
+          sudo dpkg --add-architecture i386
+          sudo apt update
+          sudo apt install -y autoconf automake libtool make pkg-config mingw-w64 mingw-w64-tools \
+            libz-mingw-w64-dev wine32 wine64 wget unzip lcov gcovr
+          sudo update-alternatives --set x86_64-w64-mingw32-g++ /usr/bin/x86_64-w64-mingw32-g++-posix
+
+      - name: Initialize fresh Wine prefix
+        run: |
+          rm -rf "$WINEPREFIX"
+          wineboot --init || true
+
+      - name: Bootstrap
+        run: ./bootstrap
+
+      - name: Configure with MinGW and coverage flags
+        run: |
+          ./configure --host=x86_64-w64-mingw32 --target=x86_64-w64-mingw32 \
+            --disable-java --enable-shared --disable-static \
+            CFLAGS="--coverage" LDFLAGS="--coverage"
+
+      - name: Show config file
+        run: cat tsk/tsk_config.h
+
+      - name: Unpack and List the test data
+        run: |
+          cd ..
+          wget -q https://digitalcorpora.s3.amazonaws.com/corpora/drives/tsk-2024/sleuthkit_test_data.zip
+          unzip -o sleuthkit_test_data.zip
+          cd sleuthkit_test_data
+          make unpack
+          find . -ls | grep -v '[.]git'
+          echo "SLEUTHKIT_TEST_DATA_DIR=$(pwd)" >> $GITHUB_ENV
+          cd ../sleuthkit
```

Comment:
Is it necessary to get this a second time? Didn't we get it here:

https://github.com/sleuthkit/sleuthkit/blob/7f958d9f81bac643f3b562bc4b1d5ba9981cf18e/.github/workflows/code-coverage.yml#L35-L45